src/libsyntax/ext/tt/macro_rules.rs

```diff
@@ -846,8 +847,8 @@ fn is_in_follow(tok: &quoted::TokenTree, frag: &str) -> Result<bool, (String, &'
                 TokenTree::MetaVarDecl(_, _, frag) if frag.name == "block" => Ok(true),
                 _ => Ok(false),
             },
-            "ident" | "lifetime" => {
-                // being a single token, idents and lifetimes are harmless
+            "ident" | "lifetime" | "literal" => {
+                // being a single token, idents, literals and lifetimes are harmless
```

Comment:
Hmm, literals are no longer necessarily single-token, but it shouldn't change anything with regards to follow set.